index.html

```diff
@@ -260,5 +265,82 @@ <h1>Innovation IT Solution for your Business & Startup</h1>
 
     </section>
 
+    <!-- Footer Section -->
+    <footer>
+        <div class="row">
+            <div class="col1 col-lg-4">
+                <div class="icon">
+                    <div>
+                        <img src="./assets/imgs/logo-icon.png" />
+                    </div>
+                    <p>IT-KOL</p>
+                </div>
+
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Consectetur, sunt.</p>
+
+                <h3>10 (87) 738-3940</h3>
+
+                <a><p>contact@it-kil.com</p></a>
+            </div>
+
+            <div class="item col2 col-lg-2">
+                <p class="heading">Navigation</p>
+                <ul>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Services</a></li>
+                    <li><a href="#">Blog</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+
+            </div>
+
+            <div class="item col3 col-lg-2">
+                <p class="heading">Services</p>
+                <ul>
+                    <li><a href="#">Drone Mapping</a></li>
+                    <li><a href="#">Real State</a></li>
+                    <li><a href="#">Commercial</a></li>
+                    <li><a href="#">Construction</a></li>
+                </ul>
+            </div>
+
+            <div class="item col4 col-lg-4">
+                <p class="heading">Subscribe newletter</p>
+                <p>Subscribe our newletter to get updates about our services and offers</p>
+                
+                <div class="subscribe-field">
+                    <input type="text" name="email" id="email" placeholder="Email Address" />
+                    <i class="fa-solid fa-arrow-right"></i>
+                </div>
+
+                <div class="social-logos">
+                    <a href="">
+                        <i class="fa-brands fa-facebook fa-xl"></i>
+                    </a>
+
+                    <a href="">
+                        <i class="fa-brands fa-instagram fa-xl"></i>
+                    </a>
+
+                    <a href="">
+                        <i class="fa-brands fa-linkedin-in fa-xl"></i>
+                    </a>
+
+                    <a href="">
+                        <i class="fa-brands fa-youtube fa-xl"></i>
+                    </a>
+ 
+                </div>
+            </div>
+        </div>
+        
+        <p class="copyright">
+            Copyright &copy 2021 All rights reserved | This template is made by 
+            <i class="fa-solid fa-heart" style="color: #f8458a;"></i> 
+            by<span>Colorlib</span>
+        </p>
+    </footer>
+
 </body>
 </html>
```

Comment:
Add a new line here to avoid this symbol at end
<img width="63" alt="Screenshot 2023-07-07 at 6 15 16 AM" src="https://github.com/HuniaNadeem-DevIntern/Frontend-Assignment-1/assets/100589984/ba24ba50-b718-48d6-b14b-80c695ff7090">